### ISSUE TITLE
CLI: Columns are too narrow when Console is not a tty

### ISSUE THREAD

**Issue by ybyzek** [confluentinc/ksql#4141]
**Describe the bug**
Formatting has changed between 5.3.1 and 5.4.0 such that the output from a query captured using `heredoc` is very hard to read.

5.3.1 format:

```
ksql> 1576506918998 | User_2 | User_2 | Page_24 | Region_3 | FEMALE
1576506919070 | User_2 | User_2 | Page_26 | Region_3 | FEMALE
1576506919425 | User_8 | User_8 | Page_92 | Region_9 | FEMALE
1576506919480 | User_8 | User_8 | Page_95 | Region_9 | FEMALE
1576506919936 | User_8 | User_8 | Page_15 | Region_9 | FEMALE
```

5.4.0 format:

```
ksql> +-----+-----+-----+-----+-----+-----+
|ROWTI|ROWKE|USERI|PAGEI|REGIO|GENDE|
|ME   |Y    |D    |D    |NID  |R    |
+-----+-----+-----+-----+-----+-----+
|15765|User_|User_|Page_|Regio|FEMAL|
|06111|7    |7    |73   |n_8  |E    |
|108  |     |     |     |     |     |
|15765|User_|User_|Page_|Regio|FEMAL|
|06111|3    |3    |89   |n_7  |E    |
|155  |     |     |     |     |     |
|15765|User_|User_|Page_|Regio|FEMAL|
|06111|1    |1    |29   |n_4  |E    |
|247  |     |     |     |     |     |
|15765|User_|User_|Page_|Regio|FEMAL|
|06111|8    |8    |90   |n_2  |E    |
|749  |     |     |     |     |     |
|15765|User_|User_|Page_|Regio|FEMAL|
|06111|1    |1    |95   |n_4  |E    |
|802  |     |     |     |     |     |
```

**To Reproduce**

1. Observer 5.3.1 behavior: run https://github.com/confluentinc/examples/tree/5.3.1-post/cp-quickstart/start.sh and execute

```
timeout 5s ksql http://localhost:8088 <<EOF                                     
SELECT * FROM PAGEVIEWS_FEMALE LIMIT 5;             
exit ;
EOF
```

1. Observer 5.4.0 behavior: run https://github.com/confluentinc/examples/tree/5.4.x/cp-quickstart/start.sh and execute

```
timeout 5s ksql http://localhost:8088 <<EOF                                     
SELECT * FROM PAGEVIEWS_FEMALE EMIT CHANGES LIMIT 5;             
exit ;
EOF
```

**Comment by colinhicks** [comment]
Updated issue title to indicate the broader surface area of the bug. For example, this also affects spool output when the cli is run in a Docker container without the `-t` flag.

Based on my read of the code, the symptom occurs when the terminal width is reported as 0 and `MIN_CELL_WIDTH = 5` applies.

In other words, when there's no tty to attach to, I think JLine creates a virtual terminal with a column width of zero. Seems like we could increase the readability of the output with one of the following measures:
- increase `MIN_CELL_WIDTH`
- in the case where `terminal.getWidth()` returns 0, set a reasonable default.
- directly detect this case and set a reasonable default by inspecting the JLine terminal instance (I did not see a clear way to do this looking at the JLine implementation or docs)

**Comment by colinhicks** [comment]
@ybyzek, I merged 42282cd into the 5.5.x branch. This can be leveraged to create more readable output by adding `SET CLI COLUMN-WIDTH 15` at the top of the quickstart's `ksql.commands`, for example

**Comment by ybyzek** [comment]
@colinhicks you're awesome, thanks for the info!

**Comment by colinhicks** [comment]
Addressed in 42282cd
